.travis.yml

```diff
@@ -23,51 +23,13 @@ env:
     - CI_CONFIG_QUIET=true
   matrix:
     - BUILD_TYPE=default
-    - BUILD_TYPE=default-Werror
-#   - BUILD_TYPE=cmake
-#   - BUILD_TYPE=android
-#   - BUILD_TYPE=check-py
-
-#matrix:
-#  include:
-#  - env: BUILD_TYPE=valgrind
-#    os: linux
-#    dist: trusty
-#    sudo: required
-#    addons:
-#      apt:
-#        packages:
-#        - valgrind
-#        - libmagic-dev
 
 addons:
   apt:
-    sources:
-    - sourceline: 'deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_14.04/ ./'
-      key_url: 'http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_14.04/Release.key'
     packages:
     - coreutils
     - git
     - asciidoc
-    - xmlto
```

Comment:
I think `xmlto` might be needed for docs generation, and SASL tools package (and corresponding user account) for several other aspects of the system setup... I saw that the CI passed though, so not fully certain now - at least not that it strongly impacts Travis runs in particular.

Reply:
There is no explicit mention of 'xmlto' in the sources, at least.